soroban-test-wasms/wasm-workspace/Cargo.toml

```diff
@@ -18,6 +18,7 @@ resolver = "2"
 
 members = [
     "add_i32",
+    "u128_ops",
```

Comment:
Did you forget to add this directory?

Reply:
Almost! I forgot to _exclude_ this hunk from the PR. It can be added later once the SDK is updated, but it can't actually work yet. Will fix.